config/routes.rb

```diff
@@ -4,19 +4,24 @@
 
   # Sufia generated routes
   blacklight_for :catalog
-  devise_for :users
+
+  scope "/admin" do
+    devise_for :users
+  end
+
   Hydra::BatchEdit.add_routes(self)
```

Comment:
I don't know.  Batch edit is already broken at the moment.  We should look at this more carefully when we fix that bug.